packages/ripple-binary-codec/src/enums/index.ts

```diff
@@ -1,164 +1,34 @@
 import * as enums from './definitions.json'
-import { SerializedType } from '../types/serialized-type'
-import { Buffer } from 'buffer/'
-import { BytesList } from '../binary'
-
-/*
- * @brief: All valid transaction types
- */
-export const TRANSACTION_TYPES = Object.entries(enums.TRANSACTION_TYPES)
-  .filter(([_key, value]) => value >= 0)
-  .map(([key, _value]) => key)
-
-const TYPE_WIDTH = 2
-const LEDGER_ENTRY_WIDTH = 2
-const TRANSACTION_TYPE_WIDTH = 2
-const TRANSACTION_RESULT_WIDTH = 1
```

Comment:
These constants have been moved to `constants.ts` so they can be imported by the split out files that used to make up `enums/index.ts`. https://github.com/XRPLF/xrpl.js/blob/pass-through-encode/packages/ripple-binary-codec/src/enums/constants.ts